### DISCUSSION TITLE
Drag and Drop Image

### DISCUSSION THREAD

**Discussion by EnfrigFalco** [TenderOwl/Frog#68] [Ideas]
Not sure if implemented. I run Fedora 36. Dragging and dropping images and pdf's would be great. At the moment it just says 'whoops' so not sure if bugged feature, future feature etc. I am guessing it is made with Gnome in mind and I see this as a good integration feature.

Love the app, love its simplicity, the logo. Very good, very useful.

**Comment by amka** [comment]
Hey! Thank you for such kind words!
The problem with d-n-d is about Flatpak permissions. You could fix it by adding read access to user's files in Flatseal app.

I will add it as defaults in the next release, but it will take time because of moving to another place to live :(
